### PR TITLE
fix: there is only one page data

### PR DESCRIPTION
**Pull request by theowenyoung**
Hi,
   It seems to occur some errors when the instagram only have one page data, there will be no `response.data.paging.next`,  so if I set the `MaxPosts` option, it will make something wrong.

I made a simple fix, can you review it?

Thanks.

### CODE REVIEW

File: src/instagram.js [hunk 1]
```diff
@@ -101,8 +101,7 @@ export async function apiInstagramPosts({
       const results = []
       results.push(...response.data.data)
       while (
-        response.data.paging.next ||
-        (maxPosts && results.length <= maxPosts)
+        maxPosts ? (response.data.paging.next && results.length <= maxPosts) : response.data.paging.next
```
Comment (oorestisime) [inline src/instagram.js:106]: can you also please add a comment about this logic so its easier to get back to it? 
otherwise looking great and thanks!

Comment (theowenyoung) [reply src/instagram.js:106]: Ok, I will.

Comment (theowenyoung) [reply src/instagram.js:106]: Hi, @oorestisime ,

I have added some comments at there. Could you please review it? :)

**Comment by oorestisime** [comment]
Thanks for the contribution!

**Pull request merged**